src/app/app-modules/associate-anm-mo/video-consultation/video-consultation.component.html

```diff
@@ -8,17 +8,17 @@
       <mat-icon>close</mat-icon>
     </button>
 
-    <h2 id="videoConsultationTitle">Video Consultation Consent</h2>
+    <h2 id="videoConsultationTitle" class="title">Video Consultation Consent</h2>
     <p>
-      By agreeing, you consent to a secure video consultation,
+      By agreeing, your consent to a secure video consultation,
       understanding that your data will be handled confidentially.
     </p>
 
     <div class="consent-actions">
-      <button class="btn btn-success" (click)="handleConsent(true)">
+      <button class="buttonColorGreen  buttonWidth" mat-raised-button (click)="handleConsent(true)" [disabled]="videoService.videoConsultationAvailable">
```

Comment:
the class naming format is bit inconsistent, for eg line no. 17 its consent-actions and previous code btn-success are in kebab-case but you are camelCase, please use the same naming format